tests/docs/running-perf-tests.md

```diff
@@ -58,10 +58,22 @@ make docker-push
 make docker-deploy-k8s
 ```
 
+### Register app configurations
+
+```bash
+make setup-app-configurations
+```
+
+### Optional: Disable tracing
```

Comment:
The reason why tracing is uses in the text, is to make it clear that currently this only applies to tracing, but the option will eventually account for telemetry as a whole.